code/index.html

```diff
@@ -1,12 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Project 4: Pizzabot</title>
-  </head>
-  <body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="./pizza.css" />
+  <link href="https://fonts.googleapis.com/css?family=Lobster|Lobster+Two&display=swap" rel="stylesheet">
+  <title>Project 4: Pizza bot</title>
+</head>
+
+<body>
+  <div class="content">
     <h1>This is your Pizza Bot</h1>
-  </body>
-</html>
+    <h2>
+      <div id="welcome"></div>
+    </h2>
+    <button class="myButton" onclick="clicked()">Start Your Order</button>
```

Comment:
It would be nice if the function name is describing what will happen in that function